js/main.js

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable no-alert */
```

Comment:
**Can all inline eslint rule be removed, please define eslint rule in the eslint file.**

Reply:
done